main.tf

```diff
@@ -5,13 +5,6 @@ provider "triton" {
     # key_id = "${var.triton_key_id}"
 }
 
-#
-# Details about all deployments of this application
-#
-data "triton_network" "public" {
```

Comment:
I'd like to consolidate this with the other vars, but there's clearly something about TF I don't understand yet.